protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -1250,6 +1252,13 @@ func renderServices(services []*descriptor.Service, paths openapiPathsObject, re
 						} else {
 							desc = fieldProtoComments(reg, bodyField.Target.Message, bodyField.Target)
 						}
+						if b.HTTPMethod == "PATCH" && reg.GetAllowPatchFeature() {
+							for _, requestTypeField := range meth.RequestType.Fields {
+								if requestTypeField.GetTypeName() == ".google.protobuf.FieldMask" {
+									notVisibleFields[requestTypeField.GetJsonName()] = nil
```

Comment:
I think we should also only match fields that are named `update_mask`, right? So we don't even necessarily need this to be a map, it can juts be a bool, right?

Reply:
Could indeed be more concise

Reply:
> I think we should also only match fields that are named `update_mask`, right? So we don't even necessarily need this to be a map, it can juts be a bool, right?
Maybe I can modify the visibility of the fields to achieve this?

Reply:
> I think we should also only match fields that are named `update_mask`, right? So we don't even necessarily need this to be a map, it can juts be a bool, right?

Maybe I can modify the visibility of the fields to achieve this?

Reply:
I'd rather we had logic to just omit the field than set a value that's not added by the user.

Reply:
Yes, that would be bad